dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -139,16 +138,90 @@ export const Card = ({
 	showAge = false,
 	discussionId,
 }: Props) => {
-	const palette = decidePalette(format, containerPalette);
-
 	const hasSublinks = supportingContent && supportingContent.length > 0;
-	const noOfSublinks = (supportingContent && supportingContent.length) || 0;
-
+	const noOfSublinks = supportingContent?.length ?? 0;
 	const isOpinion =
 		format.design === ArticleDesign.Comment ||
 		format.design === ArticleDesign.Editorial ||
 		format.design === ArticleDesign.Letter;
 
+	const cardIsHorizontal =
+		imagePosition === 'left' || imagePosition === 'right';
```

Comment:
I've asked a question about a possible corner-case for this pattern [here](https://github.com/guardian/dotcom-rendering/pull/5197/files#r898212185).